[DOCU-1924] New "Manage decK with Docker" page (#4124)
Kong/docs.konghq.com: src/deck/guides/manage-with-docker.md

---
title: Managing decK with Docker
content_type: how-to
---

If you used the `kong/deck` Docker image to install decK, you can use the Docker image to manage decK. 

## Prerequisites
decK must be installed using a [Docker image](/deck/latest/installation/#docker-image).

## Export the configuration

> Run a command similar to the following to export the `kong.yaml` file:

Reviewer: ```suggestion
Run this command to export the `kong.yaml` file:
```

```bash
docker run -i \
-v $(pwd):/deck \
kong/deck --kong-addr http://example:KONG_ADMIN_PORT --headers kong-admin-token:KONG_ADMIN_TOKEN -o /deck/kong.yaml dump
```
In this example, `kong.yaml` is generated in `$(pwd)/kong.yaml`.

## Export objects from all workspaces
Run a command similar to the following to export the objects from all the workspaces:

```bash
docker run -i \
-v $(pwd):/deck \
--workdir /deck \
kong/deck --kong-addr http://example:KONG_ADMIN_PORT:8001 --headers kong-admin-token:KONG_ADMIN_TOKEN dump --all-workspaces
```

## Reset the configuration

> Run a command similar to the following to initial Kong objects:

Reviewer: Unless initial here means something different

```bash
docker run -i \
-v $(pwd):/deck \
kong/deck --kong-addr http://example:KONG_ADMIN_PORT:8001 --headers kong-admin-token:KONG_ADMIN_TOKEN reset
```

## Import the configuration

> Run a command similar to the following to import `kong.yaml`:

Reviewer: ```suggestion
Run this command to import `kong.yaml`:
```
